SayTheirNames/Source/View/ImageWithBlurView.swift

```diff
@@ -66,9 +66,9 @@ class ImageWithBlurView: UIView {
     }
     
     // MARK: - Method
-    public func setup(_ person: Person) {
-        guard person.images.first != nil else { return }
-        frontImageView.populate(withURL: person.images[0].personURL)
-        bgImageView.populate(withURL: person.images[0].personURL)
+    public func setup(withURLString string: String?) {
+        guard let urlString = string else { return }
```

Comment:
I think we should have a placeholder, like the fist image for example instead of just leaving it empty.